apps/events-helsinki/src/common-events/components/navigation/Navigation.tsx

```diff
@@ -16,19 +22,31 @@ import styles from './navigation.module.scss';
 
 type NavigationProps = {
   page?: PageType | ArticleType;
+  menu?: Menu;
+  languages?: Language[];
 };
 
-export default function Navigation({ page }: NavigationProps) {
+export default function Navigation({ page, menu, languages }: NavigationProps) {
```

Comment:
It would be nice to provide this Navigation as  a common component, but there are those functions which has some application configurations.

Could be considered to be done somehow in a different PR.